### ISSUE TITLE
[Bug] No space left for only one request

### ISSUE THREAD

**Issue by Iamleos** [sgl-project/sglang-jax#72]
### Checklist

- [x] 1. I have searched related issues but cannot get the expected help.
- [x] 2. The bug has not been fixed in the latest version.
- [x] 3. Please note that if the bug-related issue you submitted lacks corresponding environment info and a minimal reproducible demo, it will be challenging for us to reproduce and resolve the issue, reducing the likelihood of receiving feedback.
- [x] 4. If the issue you raised is not a bug but a question, please raise a discussion at https://github.com/sgl-project/sgl-jax/discussions/new/choose Otherwise, it will be closed.
- [x] 5. Please use English, otherwise it will be closed.

### Reproduction

`python3 -u -m sgl_jax.launch_server --model-path Qwen/Qwen-7B-Chat --trust-remote-code  --dist-init-addr=0.0.0.0:10011 --nnodes=1  --tp-size=4 --device=tpu --random-seed=3 --node-rank=0 --mem-fraction-static=0.1 --max-prefill-tokens=8192 --download-dir=/tmp --dtype=bfloat16  --skip-server-warmup  --attention-backend=fa --page-size 64 --jax-precompile-decode-bs-paddings 20  --jax-precompile-prefill-token-paddings 16384`

`evalscope eval  --model Qwen-7B-Chat --api-url http://127.0.0.1:30000/v1/chat/completions --api-key EMPTY --eval-type service --datasets gsm8k --eval-batch-size 20`

### Environment

infra: per-commit-tpu-v6e-4
python: 3.10
sglang-jax: 0.0.1
